ux.symfony.com/templates/components/CodeLine.html.twig

```diff
@@ -13,7 +13,7 @@
     <pre
         {% if height %} style="--height: {{ height }}" {% endif %}
         data-clipboarder-target="source"
-    ><code {% if highlight %} data-code-highlighter-target="codeBlock" {% endif %}>{{ __code ? __code|trim|raw : code }}</code></pre>
+    ><code>{{ __code ? __code|trim|raw : code }}</code></pre>
```

Comment:
I broke the highlighting here so this needs to be fixed. Does highlight.js detect the language?